File: uri/produto/produto.scala
object Main{
    def produto( a: Int, b: Int) : Int =
        a * b

    def main( args: Array[String]){
        var a = scala.io.StdIn.readInt : Int
        var b = scala.io.StdIn.readInt : Int
        var prod = produto(a, b)
        printf("PROD = %d\n", prod )
    }
}

File: uri/produto/produto.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

    function produto( a, b )
    {
        return a * b;
    }
    
    var a = parseInt( lines.shift() );
    var b = parseInt( lines.shift() );
    console.log('PROD = ' + produto(a, b));

